.github/workflows/ci_lin.yml

```diff
@@ -143,9 +202,9 @@ jobs:
   linux-test:
     name: linux-test-${{ matrix.test_suite }}-cpu
     runs-on: test
-    needs: [linux-build-lit]
```

Comment:
Need remove the request job in github config

Reply:
Sure will remove them when the PR are merged.